include/algorithms/logistic_regression/logistic_regression_predict_types.h

```diff
@@ -218,18 +218,14 @@ class DAAL_EXPORT Result : public algorithms::classifier::prediction::Result
     template<typename Archive, bool onDeserialize>
     services::Status serialImpl(Archive *arch)
     {
-        return daal::algorithms::classifier::prediction::Result::serialImpl<Archive, onDeserialize>(arch);
+        return daal::algorithms::classifier::prediction::interface1::Result::serialImpl<Archive, onDeserialize>(arch);
     }
 };
 typedef services::SharedPtr<Result> ResultPtr;
 typedef services::SharedPtr<const Result> ResultConstPtr;
 
 } // namespace interface1
-using interface1::Parameter;
 using interface1::Input;
-using interface1::Result;
```

Comment:
Static backward compatibility will be broken due to this change. I thought interface2 should be added for logistic regression Result to reduce changes related to static backward compatibility.

Reply:
fixed by adding usings from classifier class